locale/en-us/no.voc

```diff
@@ -0,0 +1,12 @@
+no
```

Comment:
This can be removed; the resource will be resolved from neon_core (or mycroft/ovos-core)